book/src/1.welcome/README.md

```diff
@@ -1,6 +1,47 @@
 # 📖 Teams AI Library v2
 
-Teams AI Library v2 is a suite of packages used to develop on Microsoft Teams. Rebuilt from the ground up with improved developer experience in mind, create powerful agents and applications today.
+Teams AI Library v2 is a suite of packages used to develop on Microsoft Teams. Rebuilt from the ground up with improved developer experience in mind, it's never been easier to build powerful agents and applications for the hundreds of millions of people using Microsoft Teams.
```

Comment:
for the hundreds of millions of people using Microsoft Teams.

ambiguous?

for hundreds of millions of Microsoft Teams users.

Reply:
We can do that...Microsoft brand guidelines have more recently suggested we call "users" "people", unless I'm conflating that with something else

Reply:
Changed